x-pack/legacy/plugins/siem/public/components/paginated_table/index.tsx

```diff
@@ -100,15 +100,17 @@ export interface BasicTableProps<T> {
   updateActivePage: (activePage: number) => void;
   updateLimitPagination: (limit: number) => void;
 }
+type Func<T> = (arg: T) => string | number;
 
-export interface Columns<T> {
+export interface Columns<T, U = T> {
+  align?: string;
```

Comment:
where is align coming from, this prop seems to not have been there before

Reply:
align is used in the code base @XavierM and is needed as it is part of the eui table properties. If you remove it here you will get type errors such as this:

<img width="1241" alt="Screen Shot 2019-09-24 at 9 33 46 PM" src="https://user-images.githubusercontent.com/1151048/65567339-14ed9d00-df13-11e9-99c4-20254c65c6df.png">

Reply:
@XavierM this wasn't there before because we were referring to the `Columns` interface defined in `load_more_table`. @FrankHassanabad is correct that it is needed